developerportal/apps/common/tests/urls_for_tests.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
```

Comment:
Just noticed - we don't need this AND urls defined in the test module. This is a hangover from refactoring

- [ ] @stevejalim to remove this file